### ISSUE TITLE
HA2021.12: Log warning: device_state_attributes depreciated

### ISSUE THREAD

**Issue by KNXBroker** [MTrab/landroid_cloud#87]
### Environment:

OS: core-2021.12.0b0

## Configuration

N/A

## Describe the bug

There is a new warning in the logger: 
Integration implement device_state_attributes. Please report it to the custom component author.
Seems that device_state_attributes is going to be depreciated.

## Debug log

```
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:543
First occurred: 4. Dezember 2021, 20:12:35 (3821 occurrences)
Last logged: 06:45:36

Entity sensor.landroid_robbi_battery (<class 'custom_components.landroid_cloud.sensor.LandroidSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.landroid_robbi_error (<class 'custom_components.landroid_cloud.sensor.LandroidSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.landroid_robbi_status (<class 'custom_components.landroid_cloud.sensor.LandroidSensor'>) implements device_state_attributes. Please report it to the custom component author.
```

**Comment by MTrab** [comment]
Thank you for reporting, will be fixed for next release.

**Comment by MTrab** [comment]
Fixed in v1.8.1
